docs/source/api/plugin/schema-reporting.mdx

```diff
@@ -34,6 +35,9 @@ const server = new ApolloServer({
 
 </MultiCodeBlock>
 
+### Using the plugin with multiple deployment instances
+A single instance of Apollo Server running the plugin will report the current schema hash to GraphOS in a hearbeat fashion. This hash is an identifier of the schema version, not any usage data of the schema. GraphOS will not reflect a new schema version in the changelog until all Apollo Server instances (whether that is 1 or more) are reporting the same schema hash.
```

Comment:
In general, I try to favor the present tense. (See https://developers.google.com/style/tense)

```suggestion

A single instance of Apollo Server running the plugin reports the current schema hash to GraphOS in a heartbeat fashion. This hash is an identifier of the schema version and doesn't incorporate any usage data. GraphOS only reflects a new schema version in the changelog once all Apollo Server instances—whether one or more—are reporting the same schema hash.
```